### PR TITLE
vjot example  and start method

### PR DESCRIPTION
**Pull request by gintama91**
### Description

Vjot working and start method

### Image(if needed, helps for a faster review)

Screenshot_20230728_115236

### Checklist

- [x] Run tests locally
- [x] Run linter(check for linter errors)

### CODE REVIEW

File: examples/vjot.rb [hunk 1]
```diff
@@ -29,18 +29,18 @@
         @note[0] = @title.text
         load_list
       end
-      stack :width => 1.0, :height => 200, :scroll => true do
```
Comment (noahgibbs) [inline examples/vjot.rb:32]: Hm. Using a float 1.0 to mean full-width is interesting here. Do we want to keep an unmodified copy of the legacy example, not just one modified to fit our current Scarpe?

Comment (noahgibbs) [reply examples/vjot.rb:32]: I'm not worried about the whitespace changes, of course. Those are fine.

File: lacci/lib/shoes/app.rb [hunk 2]
```diff
@@ -194,6 +194,10 @@ def click(&block)
     subscription_item(shoes_api_name: "click", &block)
   end
 
+  def start(&block)
+    yield self if block_given?
+  end
```
Comment (noahgibbs) [inline lacci/lib/shoes/app.rb:199]: Hm. Shoes has a start *event* that fires the first time the slot is drawn. But this seems different. It's sort of like the Slot#app method, but not exactly? Is this being used?

Comment (gintama91) [reply lacci/lib/shoes/app.rb:199]: hm.. ah ok sorry i misread , i just ran some example and changed code acc to the error , didn't read manual correctly my bad.

Comment (gintama91) [reply lacci/lib/shoes/app.rb:199]: removing this.

File: lacci/lib/shoes/widgets/edit_box.rb [hunk 3]
```diff
@@ -2,9 +2,9 @@
 
 module Shoes
   class EditBox < Shoes::Widget
-    display_properties :text, :height, :width
+    display_properties :text, :height, :width, :margin_bottom
```
Comment (noahgibbs) [inline lacci/lib/shoes/widgets/edit_box.rb:5]: Margin-bottom would normally be added by spacing, like we do in Slot (which does the same for Stack and Flow.) Is there a reason not to do that here?

**Pull request closed without merging**